src/iexamine.cpp

```diff
@@ -974,23 +974,35 @@ void iexamine::slot_machine(player *p, map*, const tripoint&)
 void iexamine::safe(player *p, map *m, const tripoint &examp)
 {
     if (!p->has_amount("stethoscope", 1)) {
-        add_msg(m_info, _("You need a stethoscope for safecracking."));
-        return;
+        if (query_yn(_("The safe is locked. Input a random combination?"))) {
+            p->moves -= 100;
+            // one_in(30^3) chance of guessing
+            if (one_in(27000)) {
```

Comment:
It shouldn't be a new player trap, though. Confusing mechanics like that only result in people wondering how does it work.